libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtpPacket.java

```diff
@@ -233,6 +271,24 @@ public static RtpPacket parse(ParsableByteArray packetBuffer) {
       csrc = EMPTY;
     }
 
+    //Extension.
+    byte[] headerExtension;
+    byte[] extensionPayload;
+    if (hasExtension) {
+      headerExtension = new byte[HEADER_EXTENSION_SIZE];
+      packetBuffer.readBytes(headerExtension, 0, HEADER_EXTENSION_SIZE);
+      int extensionPayloadLength = (headerExtension[2] & 0xFF) << 8 | (headerExtension[3] & 0xFF);
```

Comment:
Is the & with 0xFF necessary? I think that would just return headerExtension[N]?

Reply:
Ah. I see you may have used the 0xFF mapping due to java sign handling. You could use some guava method? Like [https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/primitives/Ints.html#fromBytes(byte,byte,byte,byte)](https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/primitives/Ints.html#fromBytes%28byte,byte,byte,byte%29).
`Ints.fromBytes(0, 0, headerExtension[2], headerExtension[3])`

Reply:
Actually that I was planning to do firstly, but didn't know could I include `Ints` in file.